util/src/main/java/tc/oc/pgm/util/bukkit/Sounds.java

```diff
@@ -58,9 +64,13 @@ static Sound sound(String legacyConstant, String modernConstant, float volume, f
     // Sound.sound due to a compiler bug
     return Sound.sound(
         MISC_UTILS.getSound(
-            BukkitUtils.parse(org.bukkit.Sound::valueOf, legacyConstant, modernConstant)),
+            BukkitUtils.parse(MISC_UTILS::getSoundValue, legacyConstant, modernConstant)),
```

Comment:
At which point does this just become redundant? constant -> sound value -> sound key, may as well just do constant to key directly as a method (and then let the platform impl ensure that it exists)?